pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -86,9 +87,13 @@ type Role string
 
 // K8SSelectorConfig is Kubernetes Selector Config
 type K8SSelectorConfig struct {
-	// +kubebuilder:validation:Required
```

Comment:
nit: why we are removing this?

Reply:
I just removed it to maintain consistency, since `+required` seemed like a common notation throughout the code base.

Reply:
Read https://book.kubebuilder.io/reference/markers.html?highlight=omitempty#difference-between--optional-and--kubebuildervalidationoptional

for fields both can be applied but +kubebuilder:validation can be applied at package level as well.

Reply:
Got it, for package level validation: `+kubebuilder:validation:Optional/Required`.